media/js/firefox/hello/start.js

```diff
@@ -296,6 +314,10 @@
         // URL query param populated at template level in the view
         var incomingConversation = $main.data('incomingConversation');
 
+        // Get referrer and set Custom Variable. none is okay here.
+        var tourSource = $main.data('tourSource');
+        gaTrack(['_setCustomVar', 13, 'Hello FTU Referral', tourSource, 2]);
```

Comment:
Keep in mind the page can load when a conversation is either waiting or open too. But if you're ok with the variable being set then too, ok.